backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5737670_sys_gh19222_AD_PInstance_Para_P_String_Secured.sql

```diff
@@ -0,0 +1,90 @@
+;
+
+-- Element: P_String_Secured
+-- 2024-10-24T11:14:26.957Z
+UPDATE AD_Element_Trl SET IsTranslated='Y',Updated=TO_TIMESTAMP('2024-10-24 11:14:26.957000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Element_ID=583347 AND AD_Language='en_US'
+;
+
+-- 2024-10-24T11:14:26.994Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(583347,'en_US')
+;
+
+-- Element: P_String_Secured
+-- 2024-10-24T11:14:46.805Z
+UPDATE AD_Element_Trl SET Description='Process Parameter',Updated=TO_TIMESTAMP('2024-10-24 11:14:46.805000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Element_ID=583347 AND AD_Language='en_US'
+;
+
+-- 2024-10-24T11:14:46.807Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(583347,'en_US')
+;
+
+-- Element: P_String_Secured
+-- 2024-10-24T11:15:04.828Z
+UPDATE AD_Element_Trl SET Description='Paramètre de processus', Name='Traiter la chaîne', PrintName='Traiter la chaîne',Updated=TO_TIMESTAMP('2024-10-24 11:15:04.828000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Element_ID=583347 AND AD_Language='fr_CH'
+;
+
+-- 2024-10-24T11:15:04.831Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(583347,'fr_CH')
+;
+
+-- Column: AD_PInstance_Para.P_String_Secured
+-- 2024-10-24T11:25:07.743Z
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Table_ID,CloningStrategy,ColumnName,ColumnSQL,Created,CreatedBy,DDL_NoForeignKey,Description,EntityType,FacetFilterSeqNo,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsExcludeFromZoomTargets,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRestAPICustomColumn,IsSelectionColumn,IsShowFilterIncrementButtons,IsShowFilterInline,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,PersonalDataCategory,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,589357,583347,0,10,283,'XX','P_String_Secured','(CASE WHEN Info=''********'' THEN ''********'' ELSE P_String END)',TO_TIMESTAMP('2024-10-24 11:25:07.579000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'N','Prozess-Parameter','D',0,999999,'Y','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','Y','N','N','N','N','N','N','N','N','N','N','N',0,'Process String','NP',0,0,TO_TIMESTAMP('2024-10-24 11:25:07.579000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,0)
```

Comment:
why not `(CASE WHEN ParameterName LIKE '%Password%' THEN ''********'' ELSE P_String END)`?

checking `Info=''********'' ` it's way too fragile.

Reply:
updated